saas/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: bc-saas
+description: bc-saas represents software-as-a-service in bestchains which provides quick way to create a blockchain service
+type: application
+version: v0.1
+icon: https://avatars.githubusercontent.com/u/116370569
+appVersion: v0.1
```

Comment:
`version` and `appVersion` ?
these two fileds is must required?

Reply:
version is required, appVersion is optional.
https://helm.sh/docs/topics/charts/#the-chartyaml-file

Reply:
`appVersion` filed has been deleted.